.github/workflows/ci_workflow.yml

```diff
@@ -471,7 +471,7 @@ jobs:
           - build: 5
             build-type: Release
             build-shared: 'ON'
-            build-docs: 'ON'
+            build-docs: 'OFF'
```

Comment:
Is this necessary?  (It means we would not have any tests on Windows that build docs.)

Reply:
In GitHub Action, there are frequent Chocolatey `Doxygen` download failures on Windows platform only. It then creates a build failure blocking any merge. So, the change disables the docs build for Windows and only preserves one docs build on Linux.